routes/moviesRouter.ts

```diff
@@ -1,47 +1,44 @@
-const Express = require("express");
-const router = Express.Router();
-const Db = require('../models/database');
+import { Database } from '../models/database';
+import { Router } from 'express';
+export const router: Router = Router();
 
-let database = new Db();
 
-console.log(database);
+const database = new Database();
 
 router.get('/',async (req,res) => {
-  const movies = await database.findAllMovies();
-  console.log(movies);
+  const movies: object = await database.findAllMovies();
```

Comment:
Al haber tipado el findAllMovies no necesitas tipar el objeto movies.

Por lo que quedaría como:

```suggestion
  const movies = await database.findAllMovies();
```

Y ya typescript infiere que es un Movies[]